src/CoreBundle/Bridge/Chameleon/TableEditor/DataMailProfileTableEditor.php

```diff
@@ -0,0 +1,65 @@
+<?php
+namespace ChameleonSystem\CoreBundle\Bridge\Chameleon\TableEditor;
+
+
+use ChameleonSystem\CoreBundle\Interfaces\FlashMessageServiceInterface;
+use ChameleonSystem\CoreBundle\ServiceLocator;
+use Psr\Log\LoggerInterface;
+use TCMSTableEditor;
+use Twig\Error\SyntaxError;
+use Twig\Environment;
+use Twig\Source;
+
+class DataMailProfileTableEditor extends TCMSTableEditor
+{
+    const MESSAGE_MANAGER_CONSUMER = 'DataMailProfileTableEditorMessages';
+
+    protected function DataIsValid(&$postData, $oFields = null)
+    {
+        $twig = $this->getTwigEnvironment();
+        try {
+            $twig->tokenize(new Source($postData['body'], 'body'));
+            $twig->tokenize(new Source($postData['body_text'], 'body_text'));
+        }catch (SyntaxError $e) {
+
+            $this->getLogger()->error(
+                sprintf('failed to pares body field in E-Mail Template %s', $postData['name']),
+                [
+                    'sFieldName' => 'body',
+                    'message' => $e->getMessage(),
+                    'guess' => $e->guess(),
+                    'context' => $e->getSourceContext()->getCode(),
+                    'exception' => $e,
+                ]
+            );
+
+            $this->getFlashMessageService()->addMessage(
+                self::MESSAGE_MANAGER_CONSUMER,
+                'chameleon_system.table_editor_twig_error',
```

Comment:
As far as I know we do not have this code in the database, yet. So this PR should add it with a sensible message.

Reply:
see above.